dnf-behave-tests/features/microdnf/install1.feature

```diff
@@ -0,0 +1,22 @@
+@no_installroot
+Feature: microdnf install command on packages
+
+
+Background:
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+@bz1734350
+@bz1779757
+Scenario: Install package from local repodata with local packages
+#1. local repo with local packages
+Given I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "install kernel"
+ Then the exit code is 0
+  And microdnf transaction is
+      | Action        | Package                                   |
+      | install       | kernel-core-0:4.18.16-300.fc29.x86_64     |
+      | install       | kernel-modules-0:4.18.16-300.fc29.x86_64  |
+      | install       | kernel-0:4.18.16-300.fc29.x86_64          |
+
```

Comment:
You now have one too many newline at the ends of the files... :) If configured properly, your editor should add a single newline automatically, as it is customary on linux.